src/main/webapp/js/receipt-analysis.js

```diff
@@ -83,3 +86,39 @@ function capitalizeFirstLetters(lowercasedString) {
 function createCategoryList(categories) {
   return categories.split(',').map((category) => category.trim());
 }
+
+/**
+ * Converts the formatted price back to a number when the user
+ * selects the price input.
+ */
+function convertPricetoValue(event) {
+  const value = event.target.value;
+  event.target.value = value ? convertStringToNumber(value) : '';
+}
+
+/**
+ * Converts a string value into a number, removing all non-numeric characters.
+ */
+function convertStringToNumber(string) {
+  return Number(String(string).replace(/[^0-9.]+/g, ''));
+}
+
+/**
```

Comment:
Some of these methods seem to be repeated from Basaam's page. Are we still making a shared js/css file to handle this?